nixos/modules/services/web-apps/nextcloud.md

```diff
@@ -245,18 +245,18 @@ that are managed by Nix. If you want automatic updates it is recommended that yo
 
 ## Known warnings {#module-services-nextcloud-known-warnings}
 
-### Failed to get an iterator for log entries: Logreader application only supports "file" log_type {#module-services-nextcloud-warning-logreader}
+### Logreader application only supports "file" log_type {#module-services-nextcloud-warning-logreader}
 
 This is because
 
 * our module writes logs into the journal (`journalctl -t Nextcloud`)
 * the Logreader application that allows reading logs in the admin panel is enabled
   by default and requires logs written to a file.
 
-The logreader application doesn't work, as it was the case before. The only change is that
```

Comment:
I think this sentence doesn't fit here anymore, does it?

Reply:
I'm not sure what you mean? I removed the sentence, so yes it doesn't fit anymore.

Reply:
If I'm reading the diff right, "The logreader application doesn't work, as it was the case before." is still there. And I don't think it really fits (anymore).

Reply:
I still don't understand what you want me to change. Can you either explain again in a different way or directly push to this PR if you don't mind?